### ISSUE TITLE
[Question] How to change Stopping Criterion for Adam optimizer

### ISSUE THREAD

**Issue by mohammad-saber** [pytorch/botorch#927]
Hi, 

Thank you for reading my question.

I am trying Adam optimizer when fitting model as below:

```
model = SingleTaskGP(X, Y, covar_module=kernel)
mll = ExactMarginalLogLikelihood(model.likelihood, model)
fit_gpytorch_model(mll, optimizer = fit_gpytorch_torch)  
```

During training, I see the message like below:

```
Iter 10/100: 1.3925570860807284
Iter 20/100: 1.2582354078728433
Iter 30/100: 1.0357007765390658
```

I checked the source code. There is a stopping criterion. 

```
    stopping_criterion = ExpMAStoppingCriterion(
        **_filter_kwargs(ExpMAStoppingCriterion, **optim_options)
    )
```

I would like to know how I can set options like `rel_tol`  to continue optimization until `maxiter`.
In other words, I want optimization to continue until maximum iteration `maxiter` (without stopping criterion).

**Comment by wjmaddox** [comment]
You can overload the `rel_tol` option in `fit_gpytorch_torch` and `fit_gpytorch_model` like this:

```python
fit_gpytorch_torch(mll, options={"rel_tol": -1});
```

**Comment by mohammad-saber** [comment]
Thank you. 

I would like to confirm how to set learning rate & maxiter. It seems that the following code is correct:

`fit_gpytorch_model(mll, optimizer = fit_gpytorch_torch, options={"rel_tol": -1, "maxiter": 250, "disp": False, "lr": 0.1}) 
`

**Comment by Balandat** [comment]
Yes, this is correct.
